insights/parsers/freeipa_healthcheck_log.py

```diff
@@ -0,0 +1,23 @@
+"""
+FREEIPA HealthCheck log
+================================================
+
+"""
+
+from .. import parser
+from insights.specs import Specs
+from insights import JSONParser
+
+
+@parser(Specs.freeipa_healthcheck_log)
+class FreeIPAHealthCheckLog(JSONParser):
+    """Class to parse /var/log/ipa/healthcheck/healthcheck.log"""
+
+    @property
+    def get_errors(self):
+        """ dict: Returns a list of errors in healthcheck logs."""
+        errors = []
+        for entry in self.data:
+            if entry['severity'] > 0:
+                errors.append(entry)
+        return errors
```

Comment:
Your docstring indicates the return type is a `dict` but it is actually a `list`.

Also please consider simplifying this as a list comprehension like:
```
return [entry for entry in self.data if entry['severity'] > 0]
```

Reply:
Updated, thank you.